quickstarts/android/03-sessionhandling.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Auth0 Android Quickstarts - 3. Session Handling
+description: This tutorial will show you how to use Lock10 to mantain a session connected.
+name: Android
+alias:
+  - Android
+  - Session Handling
+  - Quickstart
+  - Mobile 
+language:
+  - Android
+hybrid: false
+image: /media/platforms/android.png
+tags:
+  - quickstart
+  - Session Handling
+alias:
+  - android
+seo_alias: android
+---
+
+## Android - Session Handling Tutorial
+
+This tutorial will show you how to use Lock10 to mantain an active session with Auth0.
+
+The user's Credentials, contains three kind of tokens:
+
+* idToken: Identity Token with user information.
```

Comment:
I think it would be useful to clarify: "idToken (also referred to as [JWT Token](https://en.wikipedia.org/wiki/JSON_Web_Token))", I've seen it like that in several places, we can avoid confusions by being clear about it.
I'm not 100% sure whether idToken === JWT Token but I'm almost convinced. @alemata , would you confirm this?